tests/ImageTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+namespace Tests;
+
+use App\Models\User;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Str;
+use Intervention\Image\Facades\Image;
+
+/**
+ * - /images:post
+ * - /images/{id}/get
+ */
+class ImageTest extends TestCase {
+
+    public function imageProvider() {
+        return [
+            // Upload Size, Medium Size excepted, Small size excepted
+            '1440x900'  => [[1440, 900], [1440, 900], [400, 250]],
+            'small' => [[200, 200], [200, 200], [200, 200]],
+            '500x500' => [[500, 500], [500, 500], [400, 400]],
+            'wide' => [[2880, 900], [1440, 450], [400, 125]],
+            'tall' => [[900, 2880], [450, 1440], [125, 400]]
+        ];
+    }
+
+    /**
+     * upload のテスト
+     * @dataProvider imageProvider
+     */
+    public function testUpload($upload) {
+        $user = User::factory()->create();
+        [$width, $height] = $upload;
+        $this->actingAs($user)->call(
+            'POST',
+            '/images',
+            [],
+            [],
+            [
+                'content' => UploadedFile::fake()->image('hoge.jpg', $width, $height)
+            ]
+        );
+        $this->assertResponseStatus(201);
+
+        $this->seeJsonStructure(['id']);
+    }
+
+    /**
+     * LoginしてないときはUploadができない
```

Comment:
スペースの有無、見てて気にならないんですか？

```suggestion
     * Login してないときは Upload ができない
```